spec/fixtures/proofing/aamva/requests/verification_request.xml

```diff
@@ -31,7 +31,7 @@
           <nc:LocationStateUsPostalServiceCode>VA</nc:LocationStateUsPostalServiceCode>
           <nc:LocationPostalCode>20176</nc:LocationPostalCode>
         </aa:Address>
-      </dldv:verifyDriverLicenseDataRequest>
+      <aa:DocumentCategoryCode>1</aa:DocumentCategoryCode></dldv:verifyDriverLicenseDataRequest>
```

Comment:
"This looks misformatted!," you might argue. I totally agree! But this is what the code generates.